### PR TITLE
Playing with some switches

### PR DESCRIPTION
**Pull request by EranAvidor**

### CODE REVIEW

File: routes/leds.js [hunk 1]
```diff
@@ -23,11 +28,18 @@ router.get('/off', function(req, res, next) {
...
 
-router.get('/toggle', function(req, res, next) {
-  var ledStatus = greenLed.readSync();
-  ledStatus = -(ledStatus-1)
-  greenLed.writeSync(ledStatus);
-  res.send('okay');
+router.get('/:id/toggle', function(req, res, next) {
+  var color = req.params.id;
+  console.log(color + 'Led is toggeling');
+  //TODO: create method getLedVariableByColor, should return (err, ledVariable)
```
Comment (MeitalT) [inline routes/leds.js:36]: @udiudi lets do it on Tuesday :sheep:

**Comment by MeitalT** [comment]
@udiudi what do you think?

File: routes/leds.js (hunk 1 above)
Comment (MeitalT) [inline routes/leds.js]: Do you think about async method?

Comment (EranAvidor) [inline routes/leds.js:36]: @MeitalT @udiudi Do you think about async call?

Comment (udiudi) [inline routes/leds.js:36]: It's bad practice to do sync jobs in Node, since it blocks the entire process/Event Loop

File: routes/leds.js [hunk 2]
```diff
@@ -23,11 +28,18 @@ router.get('/off', function(req, res, next) {
...
+  relevantLed = global[color+'Led'];
```
Comment (udiudi) [inline routes/leds.js]: We can just call it `led`

File: routes/leds.js (hunk 1 above)
Comment (EranAvidor) [inline routes/leds.js:36]: @udiudi asked about async one..

File: routes/leds.js (hunk 2 above)
Comment (EranAvidor) [inline routes/leds.js]: @udiudi sure.

**Pull request merged**